src/test/java/POM/pages/HomePage.java

```diff
@@ -1,12 +1,28 @@
 package POM.pages;
 
+import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
 
-public class HomePage extends BasePage{
-    //Locators
+public class HomePage extends BasePage {
+    //Web Elements
+    @FindBy(xpath="//button[@data-testid='home-view-all-recently-played-btn']")
+    WebElement viewAllBtn;
+
+    @FindBy(xpath="//img[contains(@alt,'Avatar of')]")
+    WebElement myAvatarIcon;
```

Comment:
**I recommend:**  make all the locators private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

```
    @FindBy(xpath="//button[@data-testid='home-view-all-recently-played-btn']")
    private WebElement viewAllBtn;

    @FindBy(xpath="//img[contains(@alt,'Avatar of')]")
    private WebElement myAvatarIcon;
```